app/services/subject_mapper.rb

```diff
@@ -0,0 +1,239 @@
+# This is a port of https://github.com/DFE-Digital/manage-courses-api/blob/master/src/ManageCourses.Api/Mapping/SubjectMapper.cs
+class SubjectMapper
+  @ucas_further_education = ["further education",
+                             "higher education",
+                             "post-compulsory"]
+
+  @ucas_english = ["english",
+                   "english language",
+                   "english literature"]
+
+  @ucas_primary = ["early years",
+                   "upper primary",
+                   "primary",
+                   "lower primary"]
+
+  @ucas_unexpected = [
+    "construction and the built environment",
+    # "history of art",
```

Comment:
Are these comments intentional?

Reply:
Comments are intentional, I guess they have previously been deliberated. Also it's in the original code base.

Reply:
Given that there is a huge list of hardcoded list that needs to be considered for mapping/transforming/ignoring, I'm going to keep as is until we revisit this subject matter.